internal/cloud/realm/dependencies.go

```diff
@@ -18,10 +18,44 @@ const (
 	dependenciesPathPattern        = appPathPattern + "/dependencies"
 	dependenciesArchivePathPattern = dependenciesPathPattern + "/archive"
 	dependenciesDiffPathPattern    = dependenciesPathPattern + "/diff"
+	dependenciesStatusPathPattern  = dependenciesPathPattern + "/status"
 
 	paramFile = "file"
 )
 
+// DependenciesStatus is used to get information from a dependencies status request
+type DependenciesStatus struct {
+	State   string `json:"status"`
+	Message string `json:"status_message"`
+}
+
+// set of known dependencies status states
+const (
+	DependenciesStateCreated    = "created"
```

Comment:
I updated these to make them more related to the `status.State` as opposed to the `status` itself.